site/docs/tools/sql.md

```diff
@@ -4,12 +4,12 @@ Spark SQL extensions provide an easy way to execute common Nessie commands via S
 ## How to use them
 
 In order to be able to use Nessie's custom Spark SQL extensions, one needs to configure
-`org.projectnessie:nessie-spark-extensions:{{ versions.java }}` along with `org.apache.iceberg:iceberg-spark3-runtime:{{ versions.iceberg }}`.
+`org.projectnessie:nessie-spark-extensions:{{ versions.java }}` along with `org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{ versions.iceberg }}`.
```

Comment:
wouldnt it make sense to have `{{ versions.iceberg-spark }}` as well?

Reply:
Makes sense